src/tactic/simps.lean

```diff
@@ -102,7 +102,7 @@ used by the `@[simps]` attribute.
   user_attribute unit (list name × list projection_data) :=
 { name := `_simps_str,
   descr := "An attribute specifying the projection of the given structure.",
-  parser := do e ← texpr, eval_pexpr _ e }
+  parser := failed }
```

Comment:
@fpvandoorn Just checking, this isn't actually supposed to be used, right? Or is this useful for testing/developing `@[simps]`?

Reply:
This is not applied manually, also not when testing (I wasn't quite sure if there was a test that adds this, but apparently not, since the CI passes).